src/query/frontend/ast/ast.hpp

```diff
@@ -2195,17 +2195,21 @@ class IndexQuery : public memgraph::query::Query {
 
   enum class Action { CREATE, DROP };
 
+  enum class Type { LOOKUP, TEXT };
```

Comment:
Let's discuss what are all the initial types

Reply:
Ahaaa, add a code comment here explaining what is what

Reply:
Done